nym-vpn-core/crates/nym-gateway-directory/Cargo.toml

```diff
@@ -6,7 +6,7 @@ license.workspace = true
 
 [dependencies]
 async-trait.workspace = true
-chrono = "0.4.37"
+chrono = "0.4.38"
```

Comment:
why are we even using `chrono` rather than `time` like we do everywhere else?

Reply:
Good question, I have no good answer. Looking at the code it might have been introduced when lifting code from the main repo or the harbourmaster repo... yes let's add that to the todo list for changing